corehq/apps/export/det/schema_generator.py

```diff
@@ -64,7 +64,15 @@ def __init__(self, dd_property_types):
 
     @staticmethod
     def transform_path(input_path):
-        # either return hard-coded lookup or add prefix
+
+        # todo: this could be made smarter in the future for multi-parentage use cases
+        def _is_index_reference(path):
+            path_as_array = path.split('.')
+            return len(path_as_array) == 2 and path_as_array[0] == 'indices'
```

Comment:
Took me longer than it should have to figure out what this is doing.
```suggestion
            return path.startswith("indices.") and path.count(".") == 1
```